<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIEngine</class>
 <widget class="QDialog" name="UIEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,85,127)</string>
  </property>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>760</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>550</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>760</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>760</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>970</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(163,163,163)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>970</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="linePin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>801</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>152</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button0">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>970</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAnnatunnusluku">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>711</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>80</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Syötä tunnusluku</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPoistu">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>1180</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(255,255,255)}
QPushButton:pressed{background-color:rgb(163,163,163)}
QPushButton{color:black}</string>
   </property>
   <property name="text">
    <string>Lopeta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
